content/2017/08-not-all-those-who-wander-are-lost.rst

```diff
@@ -1,7 +1,7 @@
 #08 - Not all those who wander are lost
 ############################################
 
-:date: TODO
+:date: 2017-05-10
```

Comment:
Whoops, fixed.
I was working around my unfamiliarity with pelican. Switching to `:status: draft` seems the better approach.